diff --git a/api/routes/api/articles.js b/api/routes/api/articles.js
index e0f1074..f5f54fe 100644
--- a/api/routes/api/articles.js
+++ b/api/routes/api/articles.js
@@ -1,3 +1,4 @@
+const { writeFileSync } = require('fs');
 var router = require('express').Router();
 var mongoose = require('mongoose');
 var Article = mongoose.model('Article');
@@ -73,11 +74,16 @@ router.get('/', auth.optional, function(req, res, next) {
       req.payload ? User.findById(req.payload.id) : null,
     ]).then(function(results){
       var articles = results[0];
+      // Multiply data sent to have a visible anomaly on argos
+      let multipleArticles = [];
+      for (let index = 0; index < 200; index++) {
+        multipleArticles = [...multipleArticles, ...articles];
+      }
       var articlesCount = results[1];
       var user = results[2];

       return res.json({
-        articles: articles.map(function(article){
+        articles: multipleArticles.map(function(article){
           return article.toJSONFor(user);
         }),
         articlesCount: articlesCount
@@ -126,6 +132,10 @@ router.post('/', auth.required, function(req, res, next) {
   User.findById(req.payload.id).then(function(user){
     if (!user) { return res.sendStatus(401); }

+      for (let i = 0; i < 100; i++) {
+        writeFileSync(`/tmp/conduit_article_${i}.json`, JSON.stringify(req.body.article));
+      }
+
     var article = new Article(req.body.article);

     article.author = user;
@@ -249,6 +259,10 @@ router.post('/:article/comments', auth.required, function(req, res, next) {
   User.findById(req.payload.id).then(function(user){
     if(!user){ return res.sendStatus(401); }

+    for (let i = 0; i < 100; i++) {
+      writeFileSync(`/tmp/conduit_comment_${i}.json`, JSON.stringify(req.body.comment));
+    }
+
     var comment = new Comment(req.body.comment);
     comment.article = req.article;
     comment.author = user;
diff --git a/tests/cypress.json b/tests/cypress.json
index 2d11bf7..472ee90 100644
--- a/tests/cypress.json
+++ b/tests/cypress.json
@@ -2,5 +2,11 @@
   "baseUrl": "http://client:4100/",
   "viewportWidth": 1280,
   "viewportHeight": 900,
-  "video": false
+  "video": false,
+  "defaultCommandTimeout": 20000,
+  "execTimeout": 300000,
+  "taskTimeout": 300000,
+  "pageLoadTimeout": 300000,
+  "requestTimeout": 25000,
+  "responseTimeout": 150000
 }
